### PR TITLE
AreaDensity: Add g/m2, mg/m2, add operators

### LINKED ISSUE #1131: Add g/m2 and mg/m2 to AreaDensity
According to wikipedia: https://en.wikipedia.org/wiki/Area_density

> In the paper and fabric industries, it is called [grammage](https://en.wikipedia.org/wiki/Grammage) and is expressed in grams per square meter (g/m2)

According to google/wikipedia it is also unofficially abbreviated in fiber as "GSM" (gram per square meter) see https://en.wikipedia.org/wiki/Grammage

Suggestion:
* Add g/m2 to AreaDensityUnit ( https://github.com/angularsen/UnitsNet/blob/master/UnitsNet/GeneratedCode/Units/AreaDensityUnit.g.cs )
* Add "grammage" as a word to the xmldoc for AreaDensity
* As we are currently on it: Also add mg/m² as this is mentioned in the UN CEFACT 20 Standard with common code "GO"

### PR DESCRIPTION
**Pull request by Brunni**
Fixes #1131

### CODE REVIEW

**Review by angularsen — CHANGES_REQUESTED** [review]
Thank you, this looks almost ready, just a few minor comments.

File: UnitsNet.Tests/CustomCode/AreaDensityTests.cs [hunk 1]
```diff
@@ -22,13 +22,25 @@
...
         protected override bool SupportsSIUnitSystem => true;
 
+        protected override double GramsPerSquareMeterInOneKilogramPerSquareMeter => 1000;
+
         protected override double KilogramsPerSquareMeterInOneKilogramPerSquareMeter => 1;
+
+        protected override double MilligramsPerSquareMeterInOneKilogramPerSquareMeter => 1000000;
+
+        [Fact]
+        public void MassDividedByAreaEqualsAreaDensity()
```
[the remark this answers is not in the document]
Comment (Brunni) [reply UnitsNet.Tests/CustomCode/AreaDensityTests.cs]: Hi. I've add the needed unit tests. I hope it is fine.

**Comment by Brunni** [comment]
The build reached maximum of 60 Minutes. Can someone restart it?

File: UnitsNet.Tests/CustomCode/AreaDensityTests.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Brunni) [reply UnitsNet.Tests/CustomCode/AreaDensityTests.cs]: @angularsen Hi. Can you recheck, if you find some time?

**Comment by angularsen** [comment]
Sorry for the late reply, I have been missing out on email notifications for some time.

Will review soon.

**Comment by angularsen** [comment]
> The build reached maximum of 60 Minutes. Can someone restart it?
> 
> EDIT: Build seems to be normally at 58 Minutes. This is quite long. How does it come?

I have no idea, sometimes the vstest process seems to hang on AppVeyor. Other times, it is just slow. It is nowhere near as slow locally, I guess the free open source VMs have very limited capacity.
In the old days it took 4-5 minutes on AppVeyor, so something has definitely changed over time.

**Comment by angularsen** [comment]
[Release UnitsNet/4.147.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.147.0)

**Comment by angularsen** [comment]
I tried upgrading a very outdated JetBrains' dotcover tool to hopefully fix the hang/slow builds. 
It seems to help locally at least. 🤞
#1146

**Pull request merged**